Makefile

```
@@ -23,6 +24,8 @@ install: python/sentry_kafka_schemas/schema_types
 
 format:
 	black python/
+	cargo fmt
+	yarn prettier --write .
```

Is it possible to do something like:
```suggestion
	yarn prettier --write **.*.json
```

---

@lynnagara yeah I would say whatever prettier supports is fair game. It doesn't seem to touch files it doesn't understand. Unlikely it conflicts with black :D